app/components/Analyst/Project/ProjectInformation/ProjectInformationForm.tsx

```diff
@@ -179,43 +178,28 @@ const ProjectInformationForm = ({ application }) => {
         }
       }}
       isFormEditMode={isFormEditMode}
-      title="Project information"
-      schema={
-        isFormEditMode
-          ? projectInformationSchema
-          : projectInformationReadOnlySchema
-      }
+      title="Funding agreement, statement of work, & map"
+      schema={isFormEditMode ? projectInformationSchema : {}}
       theme={ProjectTheme}
-      uiSchema={
-        isFormEditMode
-          ? projectInformationUiSchema
-          : projectInformationReadOnlyUiSchema
-      }
+      uiSchema={isFormEditMode ? projectInformationUiSchema : {}}
       resetFormData={handleResetFormData}
       onSubmit={handleSubmit}
       saveBtnText="Save & Import Data"
       setIsFormEditMode={(boolean) => setIsFormEditMode(boolean)}
+      showEditBtn={false}
       hiddenSubmitRef={hiddenSubmitRef}
     >
-      <div style={{ display: 'flex' }}>
-        {!isFormEditMode && (
-          <MetabaseLink
-            href={getMetabaseLink()}
-            text="View project data in Metabase"
-            width={326}
-          />
-        )}
-        {!isFormEditMode && hasChangeRequest && (
-          <StyledChangeRequestApproved>
-            <FileHeader
-              icon={faClockRotateLeft}
-              title="Change request approved"
-            />
-            <div>These are the latest Statement of Work tables</div>
-            <div>All versions can be found in Metabase</div>
-          </StyledChangeRequestApproved>
-        )}
-      </div>
+      {!isFormEditMode && (
+        <ReadOnlyView
+          dateSigned={formData?.main?.dateFundingAgreementSigned}
```

Comment:
Should this be formatted to MMM DD, YYYY as per the design?

Reply:
Thanks, good catch! Fixed!